opentelemetry-sdk/src/trace/provider.rs

```diff
@@ -52,8 +76,11 @@ impl Default for TracerProvider {
 
 impl TracerProvider {
     /// Build a new tracer provider
-    pub(crate) fn new(inner: Arc<TracerProviderInner>) -> Self {
-        TracerProvider { inner }
+    pub(crate) fn new(inner: TracerProviderInner) -> Self {
+        TracerProvider {
+            inner: Arc::new(inner),
+            is_shutdown: Arc::new(AtomicBool::new(false)),
```

Comment:
Is there any benefit to creating another `Arc` pointer just for shutdown? Could we reuse the existing `Arc` pointer `inner` to also hold `is_shutdown` inside it?